CHANGELOG.md

```diff
@@ -81,6 +81,7 @@
 - Bugfix: Fixed hidden tooltips when always on top is active (#2384)
 - Bugfix: Fix CLI arguments (`--help`, `--version`, `--channels`) not being respected (#2368, #2190)
 - Bugfix: Fix Twitch cheer emotes not displaying tooltips when hovered (#2434)
+- Bugfix: Fix directory not opening when clicking "Open AppData Directory" setting button on macOS (#2531)
```

Comment:
I suggest also adding this PR to the entry:
```suggestion
- Bugfix: Fix directory not opening when clicking "Open AppData Directory" setting button on macOS (#2531, #2537)
```

Reply:
Resolved.